v2/kafka-to-gcs/src/main/java/com/google/cloud/teleport/v2/transforms/AvroWriteTransform.java

```diff
@@ -196,19 +250,26 @@ public WriteFilesResult<AvroDestination> writeToGCS(PCollection<GenericRecord> g
   }
 
   public static class ConvertBytesToGenericRecord
-      extends DoFn<KafkaRecord<byte[], byte[]>, GenericRecord> {
+      extends DoFn<
+          KafkaRecord<byte[], byte[]>,
+          FailsafeElement<KafkaRecord<byte[], byte[]>, GenericRecord>> {
     private transient SchemaRegistryClient schemaRegistryClient;
     private String schemaRegistryURL = null;
     private String schemaPath = null;
     private boolean useMock;
```

Comment:
Seems like all 3 can be made `final`?

Reply:
Getting error when changing this to final